From 5e66a86def6738c56f14e1faccf6ee1f4a8ef93b Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Mon, 4 Nov 2024 14:38:52 +0800
Subject: [PATCH 09/14] ARM: dts: aspeed: catalina: enable uart dma mode

enable uart dma mode in uart3.

BaseOn: https://lore.kernel.org/openbmc/20230320081133.23655-1-chiawei_wang@aspeedtech.com/
The DMA UART driver has not yet been merged upstream, so the patch is currently only in the meta layer.

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
index 226594971b749..306a228839918 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
@@ -137,11 +137,16 @@ p2v5_bmc_aux: regulator-p2v5-bmc-aux {
 	};
 };
 
+&udma {
+	status = "okay";
+};
+
 &uart1 {
 	status = "okay";
 };
 
 &uart3 {
+	compatible = "aspeed,ast2600-uart";
 	status = "okay";
 };
 
-- 
2.31.1

